bootstrap/sql/mysql/create_tables.sql

```diff
@@ -69,6 +69,14 @@ CREATE TABLE IF NOT EXISTS file (
     UNIQUE KEY `jars_UK_name_version` (name, version)
 );
 
+CREATE TABLE IF NOT EXISTS fileblob (
+    name VARCHAR(255) NOT NULL,
+    version BIGINT NOT NULL,
+    data LONGBLOB NOT NULL,
+    timestamp BIGINT,
```

Comment:
AFAIK, "timestamp" is reserved word in mysql/oracle. It is better to rename  this field.